[2024-07-02T22:33:51.858+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecomm_data_pipeline.generate_and_insert_data scheduled__2024-07-02T00:10:00+00:00 [queued]>
[2024-07-02T22:33:51.861+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecomm_data_pipeline.generate_and_insert_data scheduled__2024-07-02T00:10:00+00:00 [queued]>
[2024-07-02T22:33:51.861+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-07-02T22:33:51.865+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): generate_and_insert_data> on 2024-07-02 00:10:00+00:00
[2024-07-02T22:33:51.866+0000] {standard_task_runner.py:57} INFO - Started process 58 to run task
[2024-07-02T22:33:51.868+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ecomm_data_pipeline', 'generate_and_insert_data', 'scheduled__2024-07-02T00:10:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/ecomm_dag.py', '--cfg-path', '/tmp/tmprfq3i47v']
[2024-07-02T22:33:51.868+0000] {standard_task_runner.py:85} INFO - Job 50: Subtask generate_and_insert_data
[2024-07-02T22:33:51.888+0000] {task_command.py:410} INFO - Running <TaskInstance: ecomm_data_pipeline.generate_and_insert_data scheduled__2024-07-02T00:10:00+00:00 [running]> on host 4a2fcc95fd99
[2024-07-02T22:33:51.922+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ecomm_data_pipeline' AIRFLOW_CTX_TASK_ID='generate_and_insert_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T00:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T00:10:00+00:00'
[2024-07-02T22:43:13.869+0000] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: ecomm_data_pipeline.generate_and_insert_data scheduled__2024-07-02T00:10:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2024-07-02T22:43:13.869+0000] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: ecomm_data_pipeline.generate_and_insert_data scheduled__2024-07-02T00:10:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2024-07-02T22:43:13.870+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
